steps/representative/representative-details/RepresentativeDetails.js

```diff
@@ -1,15 +1,56 @@
-const { Question } = require('@hmcts/one-per-page');
+'use strict';
+
+const { Question, form, field, goTo } = require('@hmcts/one-per-page');
+const { postCode, firstName, lastName, whitelist, phoneNumber } = require('utils/regex');
+const Joi = require('joi');
 const urls = require('urls');
 
+const emailOptions = { minDomainAtoms: 2 };
```

Comment:
We have already defined this here: https://github.com/hmcts/submit-your-appeal/blob/d3f535001da88944329b926c907bb338a0dd2c52/steps/identity/appellant-details/AppellantDetails.js#L6
Can we pull this out into it's own file and require it in both places. Cheers.